### PR TITLE
✨ Add Media Uploads Model

### PR DESCRIPTION
**Pull request by MeganSteenkamp**
Changes:
* Add Markdoc to author content.
* Add Media Uploads model.

Markdoc ADR: https://www.notion.so/centrapay/Use-Markdoc-to-author-Centrapay-Docs-content-1a9be0b4f3a641068e9b4a97db7ccf50

[📖 Migrate API Documentatinon](https://www.notion.so/centrapay/Migrate-API-Documentation-c6d64de29a2245358a5e9b7eed5edbcc?pvs=4)

Test plan: Expect to see media upload properties in the development environment.

image

### CODE REVIEW

File: markdoc.config.json
Comment (MeganSteenkamp) [inline markdoc.config.json:1]: This is recommended by Stripe to give VS Code language support with the Markdoc extension: https://marketplace.visualstudio.com/items?itemName=Stripe.markdoc-language-support

Comment (tom-blake) [reply markdoc.config.json:1]: @MeganSteenkamp Does the extension not work without this?

File: src/content/api/media-uploads.mdoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Media Uploads
+description: Media Uploads API Reference
+draft: true
+nav:
+  path: API
+  order: 7
+---
+
+## Model
```
Comment (tom-blake) [inline src/content/api/media-uploads.mdoc:10]: @MeganSteenkamp Are we going to add this heading to the sidebar?

File: markdoc.config.json
Comment (MeganSteenkamp) [reply markdoc.config.json:1]: @tom-blake I have syntax highlighting either way and before I had the extension a change to the configuration file loaded anyway. 
Perhaps it's not needed?

File: src/content/api/media-uploads.mdoc (hunk 1 above)
Comment (MeganSteenkamp) [reply src/content/api/media-uploads.mdoc:10]: @tom-blake yes, thanks.
Have made a separate [task](https://www.notion.so/centrapay/Render-H2-headings-in-sidebar-9baa89eca1274946b12516b67de306a4?pvs=4) for this.

**Review by AndyClifford — APPROVED** [review]

File: markdoc.config.json
Comment (MeganSteenkamp) [reply markdoc.config.json:1]: Have removed the file, can add it back if required 🤠

**Review by tom-blake — APPROVED** [review]

**Pull request merged**